### DISCUSSION TITLE
#max-updates since sync: 20 @ 3.30737e-06

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#920] [Q&A] [answered]
when I run a simulation,I use `clusteredLTS = 1,SurfaceOutputInterval = 0.05`, but in the output information I get `#max-updates since sync: 20 @ 3.30737e-06`, why this happened?
![image](https://github.com/SeisSol/SeisSol/assets/114474894/e9672afd-1775-4063-a30a-ebd40a7a1f01)
![image](https://github.com/SeisSol/SeisSol/assets/114474894/31f230e4-4a23-4c3c-b02c-554dc61b112c)
![image](https://github.com/SeisSol/SeisSol/assets/114474894/38ee3eb8-6e7f-409c-9f49-bfce19748816)

**Comment by sebwolf-de** [comment, marked as answer]
If SeisSol doesn't reach a Sync-Point (i.e. Output) it prints this statement in a specified interval. No worries, it seems to me that SeisSol is quite slow in your case, probably due to `ClusteredLTS=1`. But this looks perfectly normal! Once it reaches a simulation time of 0.05s, it will write the surface output.
